drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/utils/HttpClient.java

```diff
@@ -22,6 +27,7 @@ public Response get(String url, Map<String, String> headers) throws IOException
         try (CloseableHttpClient client = createHttpClient()) {
             final HttpGet request = new HttpGet(url);
             addHeaders(headers, request);
+            log.info("Sending request: " + request + " with headers: " + Arrays.asList(request.getAllHeaders()));
```

Comment:
> simple way to enable it?

:-D this is the reason why I didn't want to use a lower level... but I could use debug level or INFO when -Darquillian.debug is used